dashboard/config/foorm/forms/surveys/pd/ayw_workshop_pre_survey.0.json

```diff
@@ -0,0 +1,1369 @@
+       "text": "In-Person"
+      },
+      {
+       "value": "true",
+       "text": "Virtual"
+      }
+     ]
+    },
+    {
+     "type": "radiogroup",
+     "name": "planning_to_attend",
+     "title": " Are you planning to attend this upcoming workshop?",
+     "choices": [
+      {
+       "value": "yes",
+       "text": "Yes"
+      },
+      {
+       "value": "no",
+       "text": "No"
+      },
+      {
+       "value": "unsure",
+       "text": "Unsure / Don't Know Yet"
+      }
+     ]
+    },
+    {
+     "type": "panel",
+     "name": "csp_most_recent_unit_panel",
+     "elements": [
+      {
+       "type": "dropdown",
+       "name": "csp_unit",
+       "visibleIf": "{workshop_course} = 'CS Principles'",
+       "title": "In which unit from CS Principles is the lesson you most recently taught?",
+       "choices": [
+        {
+         "value": "unit_1",
+         "text": "Unit 1 - Digital Information"
+        },
+        {
+         "value": "unit_2",
+         "text": "Unit 2 - The Internet"
+        },
+        {
+         "value": "unit_3",
+         "text": "Unit 3 - Intro to App Design"
+        },
+        {
+         "value": "unit_4",
+         "text": "Unit 4 - Variables, Conditionals, and Functions"
+        },
+        {
+         "value": "unit_5",
+         "text": "Unit 5 - Lists, Loops, and Traversals"
+        },
+        {
+         "value": "unit_6",
+         "text": "Unit 6 - Algorithms"
+        },
+        {
+         "value": "unit_7",
+         "text": "Unit 7 - Parameters, Return, and Libraries"
+        },
+        {
+         "value": "unit_8",
+         "text": "Unit 8 - Create PT Prep"
+        },
+        {
+         "value": "unit_9",
+         "text": "Unit 9 - Data"
+        },
+        {
+         "value": "unit_10",
+         "text": "Unit 10 - Cybersecurity and Global Impacts"
+        }
+       ]
+      },
+      {
+       "type": "dropdown",
+       "name": "csd_unit",
+       "visibleIf": "{workshop_course} = 'CS Discoveries'",
+       "title": "In which unit from CS Discoveries is the lesson you most recently taught?",
+       "choices": [
+        {
+         "value": "unit_1",
+         "text": "Unit 1 - Problem Solving and Computing"
+        },
+        {
+         "value": "unit_2",
+         "text": "Unit 2 - Web Development"
+        },
+        {
+         "value": "unit_3",
+         "text": "Unit 3 - Interactive Animations and Games"
+        },
+        {
+         "value": "unit_4",
+         "text": "Unit 4 - The Design Process"
+        },
+        {
+         "value": "unit_5",
+         "text": "Unit 5 - Data and Society"
+        },
+        {
+         "value": "unit_6",
+         "text": "Unit 6 - Physical Computing"
+        }
+       ]
+      }
+     ],
+     "title": "In order for us to plan for your workshop, it's helpful to know Which lesson from {workshop_course} you've MOST RECENTLY TAUGHT (or will have taught) just prior to attending."
```

Comment:
should this be "which lesson" instead of "Which lesson"?

Reply:
fixed.